development/tools/cmd/synchronizer/syncomponent/report.go

```diff
@@ -0,0 +1,119 @@
+package syncomponent
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Report represents message about out of dates components
+type Report struct {
+	componentName string
+	message       string
+}
+
+// GetTitle returns title report - component name
+func (r Report) GetTitle() string {
+	return r.componentName
+}
+
+// GetValue returns information about out of date component
+func (r Report) GetValue() string {
+	return r.message
+}
+
+// GenerateMessage generates message about out of date components
+func GenerateMessage(components []*Component) []Report {
```

Comment:
if we return []Return, this method should be rather called GenerateReport ;)

Reply:
Doh!